src/components/ContextMenu/index.js

```diff
@@ -0,0 +1,136 @@
+import React, {
+  useState,
+  useRef,
+  useEffect,
+  useLayoutEffect,
+  useCallback,
+} from 'react';
+import styled from 'styled-components';
+import ContextMenuItem from './ContextMenuItem';
+
+function ContextMenu({
+  items,
+  mousePos,
+  parentRef,
+  isToLeft,
+  displayFocus,
+  onClick,
+}) {
+  const [isVisible, setIsVisible] = useState(false);
+  const [openToLeft, setOpenToLeft] = useState(false);
+  const [checkedItem, setCheckedItem] = useState(0);
+  const [menuRef, setRefPos] = useState(null);
+  const [menuPosition, setMenuPosition] = useState({
+    top: '',
+    left: '',
+  });
+  const elementRef = useRef(null);
+
+  useLayoutEffect(() => {
+    if (displayFocus || parentRef)
+      setRefPos(elementRef.current.getBoundingClientRect());
+  }, [displayFocus, parentRef]);
+
+  const initializeMenuPosition = useCallback(
+    (mousePos, menuRef, parentRef, isToLeft) => {
+      const windowWidth = window.innerWidth;
+      const windowHeight = window.innerHeight;
+      const offSetX = 10;
+      const offSetY = 20;
```

Comment:
Change offSet to offset, it is the same word :)

Reply:
Correct. Fixed ✌️.